pkgs/by-name/bm/bmputil/50-blackmagic.conf

```diff
@@ -0,0 +1 @@
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="1d50", ATTRS{idProduct}=="6018", GROUP="plugdev", MODE="0666"
```

Comment:
Just out of curiosity: where's this file from?

Reply:
I've had this floating around in my system for a while, don't really now where it's from

Reply:
Would be best to find out where it came from then, since we don't really want to vendor random files ourselves if we can

Reply:
Probably form [here](https://github.com/blackmagic-debug/blackmagic/blob/86704be968af4a8aa16e01a5f409e04028819391/driver/99-blackmagic-plugdev.rules#L4) would it be sensible to include the line via `pkgs.blackmagic.src` ?

Reply:
Looking at it, looks like we should install all the udev rules found inside `driver/` then